views/personal/residence.pug

```diff
@@ -0,0 +1,29 @@
+extends ../base
+include ../_includes/yesNoRadios
+
+block variables
+  -var title = 'Confirm if you are an Ontario resident'
+
+block content
+
+  h1 #{title}
+
+  div
+    p In order to be eligible for this service and in order to apply the proper provincial taxes to your file, you must have been living in (this is your <strong>primary residence</strong>) Ontario on Dec.31, 2018.
+
+
+  form.cra-form(method='post')
+    +yesNoRadios('residence', residence, 'Ontario resident on Dec.31, 2018?', errors)
+
+    div
+      details.
+        <summary><span>Help with Ontario residence</span></summary>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+
+    input#redirect(name='redirect', type='hidden', value='/personal/residence')
```

Comment:
this should redirect to the page we want to go to next (assuming a successful answer). I think I saw in your controller earlier that you want to direct to the address page?